config/locales/server.en.yml

```diff
@@ -4,3 +4,4 @@ en:
     docs_categories: "A list of category slugs to include in docs"
     docs_tags: "A list of tags to include in docs"
     docs_add_solved_filter: "Adds a filter for solved topics -- requires Discourse Solved to be installed and enabled"
+    docs_add_to_top_menu: "Adds a link to the top menu to browse to Docs"
```

Comment:
Is "browse to Docs" correct? (as opposed to e.g. "browse Docs" or "navigate to Docs")

Reply:
Navigate to Docs would be probably clearer.